<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowForm</class>
 <widget class="QMainWindow" name="MainWindowForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QVTKWidget" name="_qvtkWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>730</width>
      <height>730</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="_pointCloudTable">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>30</y>
      <width>460</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCamera">
    <property name="title">
     <string>Camera</string>
    </property>
    <widget class="QMenu" name="menuPico_Flexx">
     <property name="title">
      <string>Pico Flexx</string>
     </property>
     <addaction name="_startFlexxAction"/>
     <addaction name="_stopFlexxAction"/>
     <addaction name="_setConfidenceAction"/>
    </widget>
    <widget class="QMenu" name="menuIntel_Realsense">
     <property name="title">
      <string>Intel Realsense</string>
     </property>
     <addaction name="_startRSAction"/>
     <addaction name="_stopRSAction"/>
    </widget>
    <addaction name="menuPico_Flexx"/>
    <addaction name="menuIntel_Realsense"/>
   </widget>
   <widget class="QMenu" name="menuArduino">
    <property name="title">
     <string>Arduino</string>
    </property>
    <widget class="QMenu" name="menucommunicate">
     <property name="title">
      <string>Communicate</string>
     </property>
     <addaction name="_getNumberOfBytesAction"/>
     <addaction name="_getCharAction"/>
     <addaction name="_getArrayAction"/>
    </widget>
    <widget class="QMenu" name="menuControl_Motor">
     <property name="title">
      <string>Control Motor</string>
     </property>
     <addaction name="_controlMotorAction"/>
    </widget>
    <addaction name="menucommunicate"/>
    <addaction name="menuControl_Motor"/>
   </widget>
   <widget class="QMenu" name="menuPointCloud">
    <property name="title">
     <string>PointClouds</string>
    </property>
    <widget class="QMenu" name="menuICP">
     <property name="title">
      <string>ICP</string>
     </property>
     <addaction name="IterativeClosestPointAction"/>
    </widget>
    <addaction name="_keepPointCloudAction"/>
    <addaction name="menuICP"/>
   </widget>
   <addaction name="menuCamera"/>
   <addaction name="menuArduino"/>
   <addaction name="menuPointCloud"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="_startFlexxAction">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="_stopFlexxAction">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="_startRSAction">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="_stopRSAction">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="_getNumberOfBytesAction">
   <property name="text">
    <string>Get # Bytes Of Data</string>
   </property>
  </action>
  <action name="_getCharAction">
   <property name="text">
    <string>Get Char</string>
   </property>
  </action>
  <action name="_getArrayAction">
   <property name="text">
    <string>Get Array</string>
   </property>
  </action>
  <action name="_controlMotorAction">
   <property name="text">
    <string>Control Motor</string>
   </property>
  </action>
  <action name="_keepPointCloudAction">
   <property name="text">
    <string>Keep PointCloud</string>
   </property>
  </action>
  <action name="_setConfidenceAction">
   <property name="text">
    <string>Set Confidence</string>
   </property>
  </action>
  <action name="IterativeClosestPointAction">
   <property name="text">
    <string>PCL IterativeClosestPoint</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
